### PR TITLE
Transform eucp projection leaflet

### PR DESCRIPTION
**Pull request by geek-yang**
- Find the correct projection/coordinate system/resolutions for display via leaflet
- Implement the chosen projection in leaflet
- Tweak the projection parameters to configure the origin and correct scope of view

### CODE REVIEW

File: pages/cpm.vue [hunk 1]
```diff
@@ -62,27 +63,60 @@ export default {
...
+    //   transparent: true
+    // })
+
+    // this.map = L.map('mapid', {
+    //   crs: customCrs,
+    //   center: [52, 10],
+    //   zoom: 3,
+    //   layers: [background]
+    // }
+    // )
```
Comment (Peter9192) [inline pages/cpm.vue:107]: Let's remove the old code. That's the beauty of version control: you can always go back to previous versions

File: pages/cpm.vue [hunk 2]
```diff
@@ -92,6 +126,10 @@ export default {
       ['N', [[50.7, 1], [49.7, 26.7], [70.6, 44.1], [72.6, -9.4]]]
     ]
 
+    // domains = domains.map(([id, coordinates]) => (
+    //   [id, coordinates.map(coordinate => proj4('WGS84', 'EPSG:3995', coordinate))]
+    // ))
+
```
Comment (Peter9192) [inline pages/cpm.vue:132]:
```suggestion

```

File: pages/cpm.vue [hunk 3]
```diff
@@ -62,27 +63,60 @@ export default {
...
+      'EPSG:3995',
+      '+proj=stere +lat_0=90 +lat_ts=71 +lon_0=0 +k=1 +x_0=0 +y_0=0 +datum=WGS84 +units=m +no_defs',
+      {
+        resolutions: [22600, 11300, 5650, 2825, 1412.5, 706.25, 353.125, 176.5625, 88.28125, 44.140625],
+        origin: [-12403632.820000000298023, 17760.814940000000206],
+        bounds: L.bounds([-10403632.820, 17760.815, 12417841.080, -13082466.960])
+      })
+
+    // using default EPGS in leaflet
+    // const background = L.tileLayer('https://{s}.tile.osm.org/{z}/{x}/{y}.png', { attribution: '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors' })
```
Comment (Peter9192) [inline pages/cpm.vue:86]:
```suggestion

```

**Review by Peter9192 — APPROVED** [review]
Well done making this work! Can you tidy up a bit more?

File: pages/cpm.vue (hunk 1 above)
Comment (geek-yang) [reply pages/cpm.vue:107]: Corrected!

File: pages/cpm.vue (hunk 2 above)
Comment (geek-yang) [reply pages/cpm.vue:132]: Corrected!

File: pages/cpm.vue (hunk 3 above)
Comment (geek-yang) [reply pages/cpm.vue:86]: Corrected!

**Comment by geek-yang** [comment]
> Well done making this work! Can you tidy up a bit more?

Yes, I will clean up the mess. Thanks for your review @Peter9192

**Pull request merged**